src/text_formatter/text_formatter.test.tsx

```diff
@@ -1,23 +1,16 @@
-
-/**
- * --------------------------------------
- * @author: Ronja K.
- * ---------------------------------------
- */
-
 // IMPORTS
-import { textToListFormat, sentenceIntoWordList, replaceInputCaractersToReadablePrologCharacter, removeWhiteSpaces, checkAmountOfBrackets } from "./text_formatter";
-import { detectExpressionElements } from "./allowed_Expression_Detectors";
+import { formatExpressionElements, replaceInputCaractersToReadablePrologCharacter } from "./allowed_Expression_Detectors";
+import { checkAmountOfBrackets, removeWhiteSpaces, sentenceIntoWordList, textToListFormat } from "./text_formatter";
 
 test("splitting a single sentences into List of words: ", () => {
 	const result = sentenceIntoWordList("Hello, this is a test");
 	const expectedResult = ["Hello", "this", "is", "a", "test"];
 	expect(result).toEqual(expectedResult);
 });
 
-test("Test if the elements of a expression is detected correctly:", () => {
-	const result = detectExpressionElements("[AUNDTest[B<==>C]->DODERNOTNOTE]");
-	const expectedResult = " bracketLeft A conjunction Test bracketLeft B equivalence C bracketRight  implication D disjunction  negation  negation E bracketRight ";
+test("Test if the elements of a expression is detected and formatted correctly:", () => {
+	const result = formatExpressionElements("[AUNDTest[B<==>C]->DODERNOTNOTE]");
+	const expectedResult = "bracketLeft,A,conjunction,Test,bracketLeft,B,equivalence,C,bracketRight,implication,D,disjunction,negation,negation,E,bracketRight";
 	expect(result).toEqual(expectedResult);
 });
```

Comment:
More tests please that cover corner cases. Empty input, excessive whitespace, incorrect input. If an error is supposed to happen, assert that as well!